agent/src/softwarecontaineragent.cpp

```diff
@@ -1,14 +1,16 @@
 #include "softwarecontaineragent.h"
+#include <cstdint>
 
 SoftwareContainerAgent::SoftwareContainerAgent(
         Glib::RefPtr<Glib::MainContext> mainLoopContext
         , int preloadCount
         , bool shutdownContainers
         , int shutdownTimeout)
-    : m_mainLoopContext(mainLoopContext)
-    , m_preloadCount(preloadCount)
-    , m_shutdownContainers(shutdownContainers)
+: m_mainLoopContext(mainLoopContext)
```

Comment:
I don't believe there is a problem with this. parameters are not in the same indentation level with initializer lists. And i have checked with style guide. I couldn't find any contradiction .

Please guide me if you feel this is wrong.